tests/e2e/cucumber/environment/index.ts

```diff
@@ -74,6 +77,7 @@ Before(function (this: World, { pickle }: ITestCaseHookParameter) {
         break
     }
   })
+  await getAdminToken(state.browser)
```

Comment:
We may want to get an admin token only if API_TOKEN=true is provided, isn't it?
```suggestion
  if (config.apiToken) {
    await getAdminToken(state.browser)
  }
```